### DISCUSSION TITLE
Multiple expense items

### DISCUSSION THREAD

**Discussion by narendra-bhole** [microrealestate/microrealestate#147] [General]
Can MRE has option to add multiple expense items. Like one property can have multiple expcenses like property tax, maintainance etc.

**Comment by narendra-bhole** [comment]
Can any one comment on this.

Thanks in advance

**Reply by camelaissani** [reply]
@narendra-bhole unfortunately this functionality is not available.
Currently you can only add one expense per property in the tenant lease form.
Can you create an issue on that please?
